js/render/core/renderer.js

```diff
@@ -762,6 +763,19 @@ export class Renderer {
               gl.uniformMatrix4fv(program.uniform.LEFT_VIEW_MATRIX, false, views[0].viewMatrix);
               gl.uniformMatrix4fv(program.uniform.RIGHT_PROJECTION_MATRIX, false, views[1].projectionMatrix);
               gl.uniformMatrix4fv(program.uniform.RIGHT_VIEW_MATRIX, false, views[1].viewMatrix);
+
+              // for older browser that don't support projectionMatrix and transform on the depth data
+              gl.uniformMatrix4fv(program.uniform.LEFT_DEPTH_PROJECTION_MATRIX, false, views[0].projectionMatrix);
+              gl.uniformMatrix4fv(program.uniform.LEFT_DEPTH_VIEW_MATRIX, false, views[0].viewMatrix);
+              gl.uniformMatrix4fv(program.uniform.RIGHT_DEPTH_PROJECTION_MATRIX, false, views[1].projectionMatrix);
+              gl.uniformMatrix4fv(program.uniform.RIGHT_DEPTH_VIEW_MATRIX, false, views[1].viewMatrix);
+
+              if (view.depthdata && views[0].depthdata.projectionMatrix) {
```

Comment:
API Shape with the mixin is that this "always" exists (except for older browsers), but must be identity when aligned. I think your usage here would use identity when it should "fallback" to the view's values for an aligned view.

Reply:
Note that since we haven't shipped yet, if we want to change this, I think it's doable and now's the time to do so. I suppose I don't see a reason that we *couldn't* replicate the data from the XRView down here?

Reply:
yes, that is how I implemented it. I think the current spec reflects this.

Reply:
>  must be identity when aligned.

FYI it should *not* be identity

Reply:
https://immersive-web.github.io/depth-sensing/#ref-for-xrdepthinformation-sensor

Reply:
Having discussion in other comment thread.